htdocs/fourn/class/fournisseur.commande.class.php

```diff
@@ -382,12 +384,16 @@ public function fetch($id, $ref = '')
 				return 0;
 			}
 
+			$thirdparty = new Societe($this->db);
+			$thirdparty->fetch($obj->fk_soc);
```

Comment:
The fetch of child data must not be introduced into the fetch of an object.
You must call 

$object->fetch_thirdparties()
just after the $object->fetch()

So only the caller that need to have thirdparties loaded will load them.